components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuer.java

```diff
@@ -360,8 +361,8 @@ protected String signJWTWithECDSA(JWTClaimsSet jwtClaimsSet,
     }
 
     /**
-     * This method map signature algorithm define in identity.xml to nimbus signature algorithm format, Strings are
-     * defined inline hence there are not being used any where
+     * This method map signature algorithm define in identity.xml to nimbus signature algorithm format, Strings are.
```

Comment:
"This method map signature algorithm define in identity.xml to nimbus signature algorithm format, Strings are. " Is this period intended?

Reply:
resolve from https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1750/commits/1e9713b0f8e7c24ddc1bb20d7bd539b594266e1c